.eslintrc.yaml

```diff
@@ -375,6 +375,5 @@ rules:
       ["^\\.\\.(?!/?$)", "^\\.\\./?$"],
       # Other relative imports. Put same-folder imports and `.` last.
       ["^\\./(?=.*/)(?!/?$)", "^\\.(?!/?$)", "^\\./?$"],
-    ],
```

Comment:
A trailing comma after the last array element is at least no valid JSON (which is a subset of YAML), maybe that's the reason?

Reply:
Maybe! But it was still working, so I guess the parser ignored the issue :) Anyway, this should be better.

Reply:
Yeah, I've also seen similar issue with JSON files where the linter complains about comments but the parser accepts JSONC.